src/components/buyers/DealInventory.js

```
@@ -0,0 +1,102 @@
+            address,
+            bedrooms,
+            bathrooms,
+            sqft,
+            "image": image.asset->url,
+            "mediaLink": mediaLink
+          }`
+        );
+        const sortedDeals = data.sort(
+          (a, b) => convertToDate(b.timestamp) - convertToDate(a.timestamp)
+        );
+        debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)